### ISSUE TITLE
 TDLibError: User not found

### ISSUE THREAD

**Issue by Huuphuoc19** [airgram/airgram#92]
**Issue type:**

[x] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Airgram version:**
[x] `latest`
[x] `@next`
[x] `3.0.2` 

**Steps to reproduce or a small repository showing the problem:**
1. Listen on `updateBasicGroupFullInfo` to get list members (list user ids) 
2. Call API `getUser({ userId: xxx }` to get user info 
I can only get my own user info, but other users in the member list. 

<!--
    To answer those questions you need to put "x" inside the square brackets, for example:
    [x] question
    [ ] bug report

    Also, please format your code properly (by taking code blocks into ```ts .... ```)
!>

**Comment by jumoog** [comment]
https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1get_user.html

as normal person you query the Database

**Comment by esindger** [comment]
Hello,
Try to call [getChats](https://airgram.io/api-reference/api-methods/getChats) before the `getUser` method.

**Comment by phuocphan19** [comment]
> Hello,
> Try to call [getChats](https://airgram.io/api-reference/api-methods/getChats) before the `getUser` method.

Hello @esindger, I work by calling getChats first. 
Btw, May I know why?

**Comment by jumoog** [comment]
> > Hello,
> > Try to call [getChats](https://airgram.io/api-reference/api-methods/getChats) before the `getUser` method.
> 
> Hello @esindger, I work by calling getChats first. 
> Btw, May I know why? 

Because getUser queries the database. getChats store the chat info and the user info in the database.

**Comment by phuocphan19** [comment]
> > > Hello,
> > > Try to call [getChats](https://airgram.io/api-reference/api-methods/getChats) before the `getUser` method.
> > 
> > 
> > Hello @esindger, I work by calling getChats first.
> > Btw, May I know why?
> 
> Because getUser queries the database. getChats store the chat info and the user info in the database.

Is there any way to make it an online request @jumoog ?

**Comment by esindger** [comment]
> Is there any way to make it an online request

Nope. But it's not a problem to invoke `getChats` method one time after starting your application.
